hudi-common/src/main/java/org/apache/hudi/common/util/MarkerUtils.java

```diff
@@ -269,7 +269,7 @@ public static boolean hasCommitConflict(HoodieActiveTimeline activeTimeline, Set
     Set<String> missingFileIDs = currentInstants.stream().flatMap(instant -> {
       try {
         return HoodieCommitMetadata.fromBytes(activeTimeline.getInstantDetails(instant).get(), HoodieCommitMetadata.class)
-            .getFileIdAndRelativePaths().keySet().stream();
+            .getFileIdAndRelativePaths().values().stream().map(MarkerUtils::makerToPartitionAndFileID);
       } catch (Exception e) {
```

Comment:
Any tests here?

Reply:
I expected this test to be overridden by testHoodieClientBasicMultiWriterWithEarlyConflictDetection, but it wasn't.

And I find we use partition+fileId in AsyncTimelineServerBasedDetectionStrategy when we detect conflict, but fileId in SimpleDirectMarkerBasedDetectionStrategy. we should use partition+fileId both, all right?

I will go into this.